docs/details/commands/pmm-admin.md

```diff
@@ -14,6 +14,8 @@
 
 DATABASE:= [[MongoDB](#mongodb) | [MySQL](#mysql) | [PostgreSQL](#postgresql) | [ProxySQL](#proxysql)]
 
+`pmm-admin add --credentials-source=CREDENTIALS-SOURCE [FLAGS] [NAME] [ADDRESS]`
```

Comment:
The flag --credentials-source is available also with pmm-admin add external/external-serverless and haproxy, maybe it's better to move it to flags under every db type section?